internal/dev_server/ui/src/Flag.tsx

```diff
@@ -25,7 +25,8 @@ import { FormEvent, Fragment } from 'react';
 import { Icon } from '@launchpad-ui/icons';
 import { LDFlagValue } from 'launchdarkly-js-client-sdk';
 import { FlagVariation } from './api.ts';
-import { Box } from '@launchpad-ui/core';
+import { Box, Inline } from '@launchpad-ui/core';
+import _ from 'lodash';
```

Comment:
Recommend using `'lodash.isequal'` so that we just pull in the bits of the lib that we need.